lib/modules/modules.go

```diff
@@ -70,6 +70,21 @@ type Features struct {
 	IsUsageBasedBilling bool
 	// Assist enables Assistant feature
 	Assist bool
+	// DeviceTrust holds its namesake feature settings.
+	DeviceTrust DeviceTrustFeature
+}
+
+// DeviceTrustFeature holds the Device Trust feature general and usage-based
+// settings.
+// Requires Teleport Enterprise.
+type DeviceTrustFeature struct {
```

Comment:
Just want to confirm that this will work with @mcbattirola's changes that allow features to be read from SC instead of from the license.

Reply:
If this is what you have in mind, then it should still be alright: https://github.com/gravitational/teleport.e/blob/18d7bf2a3049ed913fd6d0cf9b1a0a85d968c850/lib/cloud/feature/feature.go#L18-L40

I'll wait a bit for Matheus to confirm.

Reply:
Yeah, this should be alright.